### PR TITLE
Add author roles for journal to metadata.yaml

### LINKED ISSUE #217: Contributor roles
For the resubmission, we need to map authors to contributor roles:

Conceptualization	
Data curation	
Formal analysis	
Funding acquisition	
Investigation	
Methodology	
Project administration	
Resources	
Software	
Supervision	
Validation	
Visualization	
Writing – original draft	
Writing – review & editing

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/greenelab/meta-review/issues/217

It's a bit challenging to determine how each category applies to our efforts. I'm not sure I was very consistent. Feel free to add additional commits.

### CODE REVIEW

**Comment by cgreene** [comment]
My contributions LGTM 👍

File: content/metadata.yaml [hunk 1]
```diff
@@ -87,3 +177,19 @@ author_info:
       - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison
       - Morgridge Institute for Research
     correspondence: true
+    roles:
+      - Conceptualization: true
+      - Data curation: true
+      - Formal analysis: true
```
Comment (agitter) [inline content/metadata.yaml]: You can flip this to `false`.  No one else has it set as `true`.  Everything else in my roles looks good.

The other authors' contributions look reasonable as well.

**Comment by vsmalladi** [comment]
LGTM

File: content/metadata.yaml [hunk 2]
```diff
@@ -66,6 +126,21 @@ author_info:
...
+      - Investigation: false
+      - Methodology: false
+      - Project administration: false
+      - Resources: false
+      - Software: true
+      - Supervision: false
+      - Validation: true
+      - Visualization: false
+      - Writing – original draft: false
+      - Writing – review & editing: false
```
Comment (agitter) [inline content/metadata.yaml]: Let's make this `true` as well.

**Comment by slochower** [comment]
LGTM

**Review by agitter — APPROVED** [review]
This draft looks good.  If any other authors want to alter their roles, please make a pull request and I'll update it in the submission system (if pre-submission) or with the editorial office (if post-submission).

**Pull request merged**